### PR TITLE
Adds Load Balancer Binds client and updates the Bind entity

### PR DESCRIPTION
**Pull request by DawidVH**
This MR adds a BindsClient for the /v2/binds endpoint. It also updates the Bind entity to make sure the properties are in line with the API response.

### CODE REVIEW

File: src/Loadbalancers/BindsClient.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        while ($page->pageNumber() < $page->totalPages()) {
+            $page = $this->getPage($page->pageNumber() + 1, $perPage, $filters);
+            $items = array_merge(
+                $items,
+                $page->getItems()
+            );
+        }
+
+        return $items;
+    }
```
Comment (rawveg) [inline src/Loadbalancers/BindsClient.php:75]: This isn't really the standard, should be using the PageItems trait and you won't need to add these methods.

**Review by ecloud-jay2 — DISMISSED** [review]
LGTM

**Review by rawveg — COMMENTED** [review]
Needs to use PageItems trait at src/Traits/PageItems.php rather than manually adding the getPage and getAll methods.

File: src/Loadbalancers/BindsClient.php (hunk 1 above)
Comment (DawidVH) [reply src/Loadbalancers/BindsClient.php]: updated to use PageItems, but changed the visibility of unsupported methods

**Review by ecloud-jay2 — DISMISSED** [review]
LGTM

**Review by ecloud-jay2 — APPROVED** [review]
LGTM

File: src/Loadbalancers/Entities/Bind.php [hunk 2]
```diff
@@ -6,16 +6,33 @@
...
  */
 class Bind extends Entity
 {
+    public function __construct($attributes = [])
+    {
+        parent::__construct($attributes);
+
+        if ($this->has('listenerId')) {
+            $this->set('listener_id', $this->get('listenerId'));
+        }
```
Comment (ecloud-jay2) [inline src/Loadbalancers/Entities/Bind.php:24]:
```suggestion
        } else if ($this->has('listener_id')) {
            $this->set('listenerId', $this->get('listener_id'));
        }
```

**Pull request closed without merging**